File: back-end/api/models/review.go
package models

type Review struct {
	ID             string `gorm:"type:varchar(100);primary_key" json:"id"`
	ReviewedUserID string `json:"reviewed_user_id"`
	Rating         int    `json:"rating"`
	Body           string `json:"body"`
	AuthorName     string `json:"author_name"`
	Timestamp      string `json:"timestamp"`
}

File: front_end_flutter/lib/models/review.dart
import 'package:json_annotation/json_annotation.dart';

part 'review.g.dart';

@JsonSerializable()
class Review {
  final String name;
  final String surname;
  @JsonKey(name: 'timestamp')
  final DateTime date;
  final double rating;
  @JsonKey(name: 'body')
  final String reviewBody;

  const Review({
    required this.name,
    required this.surname,
    required this.date,
    required this.rating,
    required this.reviewBody,
  });

  factory Review.fromJson(Map<String, dynamic> json) => _$ReviewFromJson(json);

  Map<String, dynamic> toJson() => _$ReviewToJson(this);
}
